Assets/MRTK/Core/Services/BaseService.cs

```diff
@@ -2,13 +2,14 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Diagnostics;
```

Comment:
Is this meant to use the Unity-defined `Debug.Assert`, or is this namespace correct?

Reply:
good catch, looks to have been added by visual studio. i will remove